src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -190,34 +347,98 @@ language called CML. The Fuchsia devsite contains an [overview of CML] and a
 }
 ```
 
+```txt
+fuchsia_components/
+┃  ┣━ package/
+┃  ┃  ┣━ bin/
+┃  ┃  ┃  ┗━ hello_fuchsia
+┃  ┃  ┣━ meta/
+┃  ┃  ┃  ┗━ package
+┃  ┃  ┗━ hello_fuchsia.manifest
+┗━━┻━ hello_fuchsia.cml
+```
+
 Now we can compile that CML into a component manifest:
 
 ```sh
-${SDK_PATH}/tools/${ARCH}/cmc compile hello_fuchsia.cml --includepath ${SDK_PATH}/pkg -o package/meta/hello_fuchsia.cm
+${SDK_PATH}/tools/${ARCH}/cmc compile fuchsia_components/hello_fuchsia.cml --includepath ${SDK_PATH}/pkg -o fuchsia_components/package/meta/hello_fuchsia.cm
+```
+
+**Current directory structure**
+```txt
+fuchsia_components/
+┃  ┣━ package/
+┃  ┃  ┣━ bin/
+┃  ┃  ┃  ┗━ hello_fuchsia
+┃  ┃  ┣━ meta/
+┃  ┃  ┃  ┣━ package
+┃  ┃  ┃  ┗━ hello_fuchsia.cm
+┃  ┃  ┗━ hello_fuchsia.manifest
+┗━━┻━ hello_fuchsia.cml
 ```
 
-`--includepath` tells the compiler where to look for `include`s from our CML.
-In our case, we're only using `syslog/client.shard.cml`.
+*Note: `--includepath` tells the compiler where to look for `include`s from our CML.
```

Comment:
Check that italics can be split across multiple lines

Reply:
Checked rich diff -- all looks good!